README.md

```diff
@@ -46,6 +46,16 @@ groups or DNF options.
 
 [issue 8]: https://github.com/fedora-python/tox-github-action/issues/8
 
+## Changelog
+
+Until version v0.4, this action always used the latest [fedora-python-tox](https://hub.docker.com/repository/docker/fedorapython/fedora-python-tox)
+image. Since version v34.0, the version of the action corresponds to
+the underlying Fedora and the important changes are mentioned below.
```

Comment:
I understand what you are trying to say with "corresponds to the underlying Fedora" because I know have the context but I am afraid other might not. How about:

> Since version 34.0, the first number in the version (also sometimes referred to as the "major version") represents the release of Fedora used in the image.

Reply:
Suggestion applied. Thanks!